### PR TITLE
Alien&HumanClothesCleansing

### PR DESCRIPTION
**Pull request by Snypehunter007**
Clothing Removals:

**Outfit Removals:**
- Red Wizard
- Purple Wizard
- Fake Wizard
- Marisa (Wizard/Witch)
- Star Traitor
- Gladiator
- Savage Hunter/Huntress
- Mailman
- Sexy Clown
- Sexy Mime
- Redcoat Uniform
- Monkey Suit
- Owlman Suit
- Champion's Equipment
- Infiltrator Clothes
- Fiend Clothes
- Familiar Clothes
- Ninja RIG
- Stealth RIG
- Hacker RIG
- Janitor ERT RIG
- Alt Atmos Void Suit
- Excavation Void Suit
- Exploration Void Suit
- Syndicate Green Suit
- Syndicate Orange Suit
- Syndicate Blue Suit
- Syndicate Black Suit
- Syndicate Black Green Suit
- Syndicate Black Blue Suit
- Syndicate Black Orange Suit
- Syndicate Black Red Suit
- Syndicate Black Medical Suit
- Syndicate Black Engineer Suit

**Hats:**
- Xmashat
- Caretaker Hood
- Overseer Hood
- Hastur's Hood
- Cuban Pete's Hat
- Centcom Hat
- Captain's Hat
- Captain's Cap
- Captain's Formal Cap
- Head of Personnel's Cap
- Warden's Hat
- Engineering Beret
- Deathsquad Beret
- Mirania Beret
- Nurse Hat
- Cueball
- Slime Hat
- Cat Ears
- Powdered Wig
- Grey Detective's Hat
- Bowler Hat (x2)
- Beaver Hat
- Boater Hat
- Fedora
- Trilby
- Cake Hat
- Red Cap
- Blue Cap
- Yellow Cap
- Mime's Cap
- Purple Cap
- Rainbow Cap
- Security Cap
- Shipping Cap
- Wizard's Cap
- Carp Styled Welding Helmet
- Syndicate Replica Helmet
- Swat Helmet
- NT Helmet
- PCRC Helmet
- Tactical Helmet
- Merc Helmet
- Ablative Helmet
- Gladiator Helmet
- ERT Commander Helmet
- ERT Security Helmet
- ERT Engineer Helmet
- ERT Medical Helmet
- Tactical Mirania Helmet
- Emergency Space Helmet

**Uniforms:**
- Psychiatrist's Turtleneck
- Captain's Jumpsuit
- Captain's Suit
- Captain's Dress
- Head of Personnel's Jumpsuit
- Head of Personnel's Fancy Jumpsuit
- Head of Personnel's Suit
- Head of Personnel's Dress
- Human Resources Dress
- Red Lawyer's Suit
- Blue Lawyer's Suit
- Purple Lawyer's Suit
- Nurse's Suit
- Nurse's Dress
- Geneticist's Jumpsuit
- Chemist's Jumpsuit
- Scientist's Jumpsuit
- Virologist's Jumpsuit
- Ward-Takahashi Jumpsuit
- Aether Jumpsuit
- Focal Point Jumpsuit
- Hephaestus Jumpsuit
- Warden's Jumpsuit
- Security Officer's Jumpsuit
- Dispatcher's Uniform
- Major Bill's Uniform
- Morpheus Cyberkinetics' Uniform
- Skinner Catering Uniform
- Head of Security's Jumpsuit
- Head of Security's Formal Suit
- Head of Security's Formal Dress
- Schoolgirl Uniform
- Green Dress
- Orange Dress
- Pink Dress
- Purple Dress
- Yellow Dress
- Saloon Dress
- Orange Wedding Dress
- Purple Wedding Dress
- Blue Wedding Dress
- Red Wedding Dress
- White Wedding Dress
- Sundress
- White Sundress
- Cheongsam
- Abaya
- Kimono
- Soviet Uniform
- Frontier Clothes
- Kilt
- Overalls
- Psychic Suit
- Scratch's Suit
- Vice Officer's Jumpsuit
- Rainbow Jumpsuit
- Johnny~~ Jumpsuit
- Jean Shorts
- Jean Shorts Classic
- Jean Shorts Mustang
- Jean Shorts Young Folks
- Jean Shorts Black
- Jean Shorts Grey
- Jean Shorts Khaki
- Plaid Blue Skirt
- Plaid Red Skirt
- Plaid Purple Skirt
- Khaki Skirt
- Swept Skirt
- Short Skirt

**Suits:**
- Fire (old fire suit)
- Security Armor (old)
- Detective Armor
- Caretaker Cloak
- Overseer Cloak
- Alt Cultrobes
- Psychic Purple Robes
- Magus(red) Robes
- Magus(blue) Robes
- Captain's Tunic
- Captain's Jacket
- Captain's Wintercoat
- Head of Security's Jacket
- Warden's Jacket
- Security Officer's Jacket
- Security's Wintercoat
- Atmospherics' Wintercoat
- Miner's Wintercoat
- Forensics Jackets
- First Responder's Jacket
- Chief Medical Officer's Alt Labcoat
- Green Labcoat
- Geneticist Labcoat
- Chemist Labcoat
- Virologist Labcoat
- Science Labcoat
- Xynergy Labcoat
- Surgical Apron
- Centcom Armor
- Thunderdome Armor
- Lasertag Armor
- ERT Commander Armor
- ERT Security Armor
- ERT Engineer Armor
- ERT Medical Armor
- NT Vest
- NT Webvest
- Warden Webvest
- Detective Vest
- PCRC Vest
- Press Vest
- Merc Vest
- Mirania Vest

**Gloves:**
- Rainbow Gloves

**Shoes:**
- Caretaker Shoes
- Blue Shoes
- Red Shoes
- Green Shoes
- Purple Shoes
- Yellow Shoes
- Rainbow Shoes
- Hightop Shoes
- Mime Shoes
- Athletic Shoes
- Galoshes
- Jackboots
- Unathi Jackboots
- Unathi Workboots
- Swat Boots
- Marisa Sandals
- Swimming Fins
- Medieval Boots

**Eyes:**
- Green Glasses
- SecHUD Goggles
- Thermal Monocle
- Eyepatch HUDs
- Tajaran Eyewear
- Jensen SecHud

**Masks:**
- Luchador Masks
- Tactical Balaclava
- Budget Gas Mask
- Gas Mask (old)
- Swat Mask
- Syndicate Mask
- Pig Mask
- Horsehead Mask
- Rubber Masks
- Vox Masks
- Fake Moustache
- Snorkel
- Blue Scarf
- Red Scarf
- Green Scarf
- Ninja Scarf

**Accessories:**
- Dashiki
- Thawb
- Sherwani
- Qipao
- Zhongshan
- Tangzhuang
- Bronze Medal
- Iron Medal
- Gold Medal
- Sol Gov Fleet Patches

**Additional Removals:**
- Earrings
- Old clothing sprites with no code.

Removal as a result of missing items:

**Runes:**
- Armor Rune

### CODE REVIEW

File: code/game/objects/structures/crates_lockers/closets/job_closets.dm [hunk 1]
```diff
@@ -40,7 +39,6 @@
 
 /obj/structure/closet/chefcloset/WillContain()
 	return list(
-		/obj/item/clothing/under/sundress,
```
Comment (NanakoAC) [inline code/game/objects/structures/crates_lockers/closets/job_closets.dm:43]: i dont see whats wrong with this one, service staff dressing nicely isnt too bad

File: code/modules/clothing/head/misc.dm [hunk 2]
```diff
@@ -267,26 +243,12 @@
 	flags_inv = BLOCKHAIR
 	body_parts_covered = 0
 
-/obj/item/clothing/head/hijab
```
Comment (NanakoAC) [inline code/modules/clothing/head/misc.dm:270]: whats up with this? I dont think islam is entirely gone, but there are non religious reasons to wear it too

File: code/modules/clothing/suits/miscellaneous.dm [hunk 3]
```diff
@@ -57,21 +57,6 @@
 	fire_resist = T0C+5200
 	flags_inv = HIDEGLOVES|HIDESHOES|HIDEJUMPSUIT
 
-
-/obj/item/clothing/suit/greatcoat
```
Comment (NanakoAC) [inline code/modules/clothing/suits/miscellaneous.dm:61]: whats wrong with a big coat? This sort of thing would be popular with captains

File: code/modules/clothing/under/miscellaneous.dm [hunk 4]
```diff
@@ -291,56 +233,6 @@
 	worn_state = "dress_fire"
 	body_parts_covered = UPPER_TORSO|LOWER_TORSO
 
-/obj/item/clothing/under/dress/dress_fire
```
Comment (NanakoAC) [inline code/modules/clothing/under/miscellaneous.dm:294]: theres nothing wrong with dresses, not everyone wears uniform, especially if off duty. These might be used for some roleplaying fun

File: code/modules/clothing/under/miscellaneous.dm [hunk 5]
```diff
@@ -385,67 +277,6 @@
 	item_state = "kilt"
 	worn_state = "plaid_purple"
 
-//wedding stuff
-/obj/item/clothing/under/wedding/
```
Comment (NanakoAC) [inline code/modules/clothing/under/miscellaneous.dm:389]: i dont suppose crew members might ever get married? Seems like a fun IC event for necromorphs to interrupt
even if unlikely, i'd rather keep at least one of these around

File: code/modules/clothing/under/miscellaneous.dm [hunk 6]
```diff
@@ -535,22 +366,6 @@
 	item_state = "lb_suit"
 	worn_state = "mechanic"
 
-/obj/item/clothing/under/cheongsam
```
Comment (NanakoAC) [inline code/modules/clothing/under/miscellaneous.dm:538]: see above on dresses, i dont see the harm in keeping these

File: code/modules/clothing/under/skirts.dm [hunk 7]
```diff
@@ -49,51 +49,4 @@
 	desc = "A short skirt, made of some semi-gloss material."
 	icon_state = "skirt_short"
 	worn_state = "skirt_short"
-	body_parts_covered = LOWER_TORSO
-
-// dresses
-/obj/item/clothing/under/skirt_c/dress
```
Comment (NanakoAC) [inline code/modules/clothing/under/skirts.dm:55]: more dresses doing no harm

**Review by NanakoAC — CHANGES_REQUESTED** [review]
Generally i'm cool with removing:
-Bay setting specific content, including their aliens, factions and characters
-Memes
-Silly costumes.

That covers most of this stuff, but there's a few things i've marked that don't fit into any of these categories, and keeping those wouldn't do harm to immersion or our setting. Dresses , greatcoat, hijab, etc. These are the sort of things crew might have brought along in their personal luggage and wear on special occasions. They shouldn't be removed

File: code/modules/clothing/suits/miscellaneous.dm (hunk 3 above)
Comment (Snypehunter007) [reply code/modules/clothing/suits/miscellaneous.dm:61]: It's a Nazi coat.

File: code/modules/clothing/under/miscellaneous.dm (hunk 5 above)
Comment (ErwinSnowden) [reply code/modules/clothing/under/miscellaneous.dm:389]: That seems very LRP for the grimdark attitude of the game.

File: code/game/mecha/mech_sensor.dm [hunk 8]
```diff
@@ -13,77 +13,4 @@
 	var/id_tag = null
 
 	var/frequency = 1379
-	var/datum/radio_frequency/radio_connection
```
Comment (NanakoAC) [inline code/game/mecha/mech_sensor.dm]: Don't be deleting mechas, this is unnecessary. We will probably find a use in future for their code, it is very flexible.

Disable them ingame, make sure parts and fabricators can't be made, but don't delete things

**Review by NanakoAC — CHANGES_REQUESTED** [review]
The mecha code is nono, that needs to stay. i may find use for it in future vehicles for sprawl or similar

**Review by JakeDaBoss — APPROVED** [review]
Getting rid of what's gonna go unused is good practice.

And those who defend keeping these clothes, don't know what Dead Space is about, and "casual wear" is rarely seen.

If anything, we can sprite better ones that fit our lore more.

File: code/modules/clothing/spacesuits/rig/suits/hacker.dm [hunk 9]
```diff
@@ -1,38 +0,0 @@
...
-
-/obj/item/clothing/suit/lightrig/hacker
-	siemens_coefficient = 0.2
-
-/obj/item/clothing/shoes/lightrig/hacker
-	siemens_coefficient = 0.2
-	item_flags = ITEM_FLAG_NOSLIP //All the other rigs have magboots anyways, hopefully gives the hacker suit something more going for it.
-
-/obj/item/clothing/gloves/lightrig/hacker
-	siemens_coefficient = 0
```
Comment (ErwinSnowden) [inline code/modules/clothing/spacesuits/rig/suits/hacker.dm:38]: Wasn't this intentionally re-designed by nanako and planned for integration into the patron item list?

Comment (Snypehunter007) [reply code/modules/clothing/spacesuits/rig/suits/hacker.dm:38]: Different hacker suit.

Bay has one as well.

File: code/modules/clothing/suits/jobs.dm [hunk 10]
```diff
@@ -181,36 +132,17 @@
...
-/obj/item/clothing/suit/surgicalapron
-	name = "surgical apron"
-	desc = "A sterile blue apron for performing surgery."
-	icon_state = "surgical"
-	item_state = "surgical"
 	blood_overlay_type = "armor"
-	body_parts_covered = UPPER_TORSO|LOWER_TORSO
 	allowed = list(/obj/item/stack/medical, /obj/item/weapon/reagent_containers/dropper, /obj/item/weapon/reagent_containers/hypospray, /obj/item/weapon/reagent_containers/syringe, \
-	/obj/item/device/healthanalyzer, /obj/item/device/flashlight, /obj/item/device/radio, /obj/item/weapon/tank/emergency,/obj/item/weapon/scalpel,/obj/item/weapon/retractor,/obj/item/weapon/hemostat, \
-	/obj/item/weapon/cautery,/obj/item/weapon/bonegel,/obj/item/weapon/FixOVein)
```
Comment (ErwinSnowden) [inline code/modules/clothing/suits/jobs.dm:213]: Why restrict the ability to store surgical tools on your apron?

Comment (Snypehunter007) [reply code/modules/clothing/suits/jobs.dm:213]: The Apron got removed.

**Comment by NanakoAC** [comment]
Things I wish to keep, round 3:

You asked about rigs, i said don't remove any. These allneed to be left alone, disable but don't remove.
Ninja RIG
Stealth RIG
Hacker RIG
Alt Atmos RIG
Excavation RIG
Exploration RIG
Janitor ERT RIG


Hats: These all seem either generic enough, or desireable to officers, to be worth keeping. Some for civilian use in sprawl
Captain's Hat
Captain's Cap
Captain's Formal Cap
Head of Personnel's Cap
Warden's Hat
Engineering Beret
Grey Detective's Hat
Bowler Hat (x2)
Beaver Hat
Boater Hat
Fedora
Trilby

Uniforms:
Psychiatrist's Turtleneck
Head of Personnel's Suit
Head of Personnel's Dress
Human Resources Dress

Suits:
Fire (old fire suit) //This was never meant to be removed, it'll be valueable as a crafting base
Captain's Wintercoat //Winter gear will be used for tau volantis eventually
Security's Wintercoat
Atmospherics' Wintercoat
Miner's Wintercoat
First Responder's Jacket //This is an iconic paramedic thing

Shoes: //These are all generic and popular
Athletic Shoes
Galoshes
Jackboots


Masks:
Tactical Balaclava
Blue Scarf //Theres nothing wrong with scarfs
Red Scarf
Green Scarf

**Review by NanakoAC — APPROVED** [review]

**Pull request merged**